cegs_portal/search/templates/search/v1/experiment_list.html

```diff
@@ -1,39 +1,122 @@
+        font-style: italic;
+        color: #718096;
+        margin-top: 0.5rem;
+    }
+
+    .description {
+        margin-top: 0.5rem;
+        color: rgb(87, 86, 86);
+        width: 1100px;
+    }
+
+    h1 {
+        font-family: "Arial", sans-serif;
+        font-size: 2rem;
+        color: #094474;
+        text-shadow: 2px 2px #e3f2fd;
+        margin-bottom: 10px;
+        margin-left: 40px;
+        display: flex;
+        align-items: center;
+    }
+
+    h1 svg {
+        margin-right: 10px;
+        fill: currentColor;
+        height: 1em;
+    }
+</style>
+{% endblock %} {% block title %}Experiments{% endblock %} {% block content %}
 
-{% block title %}Experiments{% endblock %}
-{% block content %}
 <div>
-    <div class="text-2xl font-bold">Experiments</div>
+    <div style="display: flex; justify-content: space-between; align-items: center">
+        <h1>
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 640 512" class="icon">
+                <!--! Font Awesome Pro 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. -->
+                <path
+                    d="M175 389.4c-9.8 16-15 34.3-15 53.1c-10 3.5-20.8 5.5-32 5.5c-53 0-96-43-96-96V64C14.3 64 0 49.7 0 32S14.3 0 32 0H96h64 64c17.7 0 32 14.3 32 32s-14.3 32-32 32V309.9l-49 79.6zM96 64v96h64V64H96zM352 0H480h32c17.7 0 32 14.3 32 32s-14.3 32-32 32V214.9L629.7 406.2c6.7 10.9 10.3 23.5 10.3 36.4c0 38.3-31.1 69.4-69.4 69.4H261.4c-38.3 0-69.4-31.1-69.4-69.4c0-12.8 3.6-25.4 10.3-36.4L320 214.9V64c-17.7 0-32-14.3-32-32s14.3-32 32-32h32zm32 64V224c0 5.9-1.6 11.7-4.7 16.8L330.5 320h171l-48.8-79.2c-3.1-5-4.7-10.8-4.7-16.8V64H384z"
+                />
+            </svg>
+            Experiment Index
+        </h1>
+    </div>
     <div>
-        <table class="data-table">
-            <tr><th>Accession ID</th><th>Name</th></th><th>Description</th><th>Cell Line(s)</th></tr>
-            {% for experiment in experiments %}
-            <tr class="{% cycle 'bg-blue-100' 'bg-blue-50' %}">
-                <td><a href="{% url 'search:experiment' experiment.accession_id %}">{{ experiment.accession_id }}</a></td>
-                <td>{{ experiment.name }}</td>
-                <td>{{ experiment.description }}</td>
-                <td>{{ experiment.cell_lines }}</td>
-            </tr>
-            {% endfor %}
-        </table>
-        <div class="pagination">
-            <span class="step-links">
-                {% if experiments.has_previous %}
+        {% for experiment in experiments %}
+        <a href="{% url 'search:experiment' experiment.accession_id %}" class="content-container-link">
+            <div class="content-container">
+                <div>
+                    <div class="name">{{ experiment.name }}</div>
+                    <div class="description">{{ experiment.description }}</div>
+                    <div class="flex justify-between cell-accession">
```

Comment:
There is no cell-accession class in the CSS.

Reply:
Ah, didnt take it out after making the class to move accession id.